new_client/src/components/about-us/AboutUs.jsx

```diff
@@ -14,10 +14,27 @@ import GenericDetails from "./GenericDetails";
 import GSes from "../assets/coc-gses.webp";
 import Members from "../assets/coc-members.webp";
 import Banner from "./Banner";
-
 import "./AboutUs.css";
+import Image from "./Image";
+import Details from "./details";
 
 export default function AboutUs() {
+  const details = Details;
+  const arr = [];
+
+  for (let i = 0; i < 30; i += 2) {
+    arr.push(
+      <div class="com-mem">
+        <Image class="member" name={details[i].name} img={details[i].img} />
+        <Image
+          class="member"
```

Comment:
For this and all other instances in this component, please update `class` to `className`.

Reply:
The Image Component is a custom component, so it's ok if I use class, right?